docs/source/nlu.rst

```diff
@@ -1,10 +1,20 @@
 Natural Language Understanding (NLU)
 ====================================
 
-The NLU component is responsible for obtaining a structured representation of text utterances. Currently, it entails intent classification, entity recognition, and user satisfaction prediction.
+The NLU component is responsible for obtaining a structured representation of text utterances. Currently, it entails dialogue acts recognition, key-value annotation, and user satisfaction prediction.
```

Comment:
Same comment as above for concepts.rst

Reply:
I added an example as in concepts.rst to avoid ambiguity.